frontend/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {styled} from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+
+const MyButton = styled(Button)({
+    background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+    border: 0,
+    borderRadius: 3,
+    boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
+    color: 'white',
+    height: 48,
+    padding: '0 30px',
+});
+
+export default function ButtonComponent(props: { name: string, onClick: () => void }) {
```

Comment:
To mozesz zrobic w ten sposob:
```
import React from "react";
import { styled } from "@material-ui/core/styles";
import Button from "@material-ui/core/Button";

const ButtonComponent = styled(Button)({
  background: "linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)",
  border: 0,
  borderRadius: 3,
  boxShadow: "0 3px 5px 2px rgba(255, 105, 135, .3)",
  color: "white",
  height: 48,
  padding: "0 30px",
});

export default ButtonComponent;
```
Wtedy masz ostylowany button a jeszcze wszystkie mozliwosci buttona.

Reply:
Wtedy w jsx uzywasz tego tak:
```
<ButtonComponent>Hello</ButtonComponent>
```
I masz jak w html i mozesz korzystac jak z <button> htmlowego.